PWGGA/GammaConvBase/AliConvEventCuts.cxx

```diff
@@ -6150,75 +6157,80 @@ void AliConvEventCuts::GetNotRejectedParticles(Int_t rejection, TList *HeaderLis
         for(Int_t j = 0; j<HeaderList->GetEntries();j++){
           TString GeneratorInList   = ((TObjString*)HeaderList->At(j))->GetString();
           if (fDebugLevel > 0 )  cout << GeneratorInList.Data() << endl;
-          if(GeneratorInList.Contains(GeneratorName) ){
-            if (fDebugLevel > 0 ) cout << "accepted" << endl;
-            if (GeneratorInList.BeginsWith("PARAM") || GeneratorInList.CompareTo("BOX") == 0 ){
-              if(fMCEvent){
-                if (fPeriodEnum == kLHC14a1b || fPeriodEnum == kLHC14a1c ){
-                  if (fDebugLevel > 2 )cout << "number of produced particle: " <<  gh->NProduced() << endl;
-                  if (fDebugLevel > 2 )cout << "pdg-code of first particle: " <<  fMCEvent->Particle(firstindexA)->GetPdgCode() << endl;
-                  if (fMCEvent->Particle(firstindexA)->GetPdgCode() == fAddedSignalPDGCode ) {
-                    if (gh->NProduced() > 10 && fMCEvent->Particle(firstindexA+10)->GetPdgCode() == fAddedSignalPDGCode && GeneratorInList.CompareTo("BOX") == 0){
-                      if (fDebugLevel > 0 ) cout << "cond 1: "<< fnHeaders << endl;
-                      fnHeaders++;
-                      continue;
-                    } else if (gh->NProduced() == 3 && GeneratorInList.Contains("PARAM_EMC") && (i == 3 || i == 5) ){
-                      if (fDebugLevel > 2 ) cout << "accepted EMC header "<< endl;
-                      if (fDebugLevel > 0 ) cout << "cond 1: "<< fnHeaders << endl;
-                      fnHeaders++;
-                      continue;
-                    } else if (gh->NProduced() > 2 && GeneratorInList.Contains("PARAM_PHOS") && (i == 4 || i == 6) ){
-                      if (fDebugLevel > 2 ) cout << "accepted PHOS header "<< endl;
-                      if (fDebugLevel > 0 ) cout << "cond 1: "<< fnHeaders << endl;
-                      fnHeaders++;
-                      continue;
+          if (fPeriodEnum==kLHC20g10){
+            if (GeneratorName.Contains(GeneratorInList)){ ++fnHeaders; }
+          }
```

Comment:
basically only this if (fPeriodEnum==kLHC20g10) gets added here. Everyting else that was already here goes untouched into the else of the if